examples/ecmascript/es5.ohm

```diff
@@ -338,18 +327,13 @@ ES5 {
                   | "!"    UnaryExpression  -- lnot
                   | PostfixExpression
 
-  MultiplicativeExpression = MultiplicativeExpression "*" UnaryExpression -- mul
-                           | MultiplicativeExpression "/" UnaryExpression -- div
-                           | MultiplicativeExpression "%" UnaryExpression -- mod
+  MultiplicativeExpression = MultiplicativeExpression ("*"|"/"|"%") UnaryExpression -- mul
```

Comment:
IMO the original version gives you a better API for writing semantic actions. (With this version, you might have to write a `switch` or chain of `if`s to figure out what to do, etc.)